source/Nevermore.IntegrationTests/IntegrationTestDatabase.cs

```diff
@@ -55,6 +55,7 @@ public IntegrationTestDatabase()
         void CreateDatabase()
         {
             ExecuteScript(@"create database [" + TestDatabaseName + "] COLLATE SQL_Latin1_General_CP1_CS_AS", GetMaster(TestDatabaseConnectionString));
+            ExecuteScript(@"CREATE TYPE dbo.[ParameterList] as TABLE([ParameterValue] nvarchar(300))", TestDatabaseConnectionString);
```

Comment:
Nevermore includes a `KeyAllocator` upgrade script, should the creation of the `ParamerterList` type also be included as an "in-the-box" script?

Reply:
Good idea, might add it as a second script